src/components/curator-actions/AccordionComponent.svelte

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+	import InfoSvg from './InfoSVG.svelte';
+
+	let bounty = {
+		id: '#88',
+		title: 'Community Event Activity Bounty'
+	};
+
+	function handleToggleClick(this: HTMLElement) {
+		const accordion = this.closest('.accordion');
+		if (!accordion) {
+			console.error('Accordion not found!');
+			return;
+		}
+		accordion.classList.toggle('expanded');
+		this.textContent = accordion.classList.contains('expanded')
+			? 'keyboard_arrow_up'
+			: 'keyboard_arrow_down';
+	}
+</script>
+
+<div class="accordion bg-secondary border-accent rounded-md p-3 my-6">
+	<div
+		class="accordion-header font-bold text-lg lg:hidden text-white flex justify-between items-center p-2 space-x-2"
+	>
+		<span class="bounty-id mb-7">{`${bounty.id}`}</span>
+		<span class="bounty-title ml-3">{`${bounty.title}`}</span>
+		<button
+			class="accordion-toggle material-symbols-outlined border-white border-2 rounded-full p-3 w-8 h-8 text-2xl flex justify-center items-center ml-3"
```

Comment:
```suggestion
			class="material-symbols-outlined border-white border-2 rounded-full p-3 w-8 h-8 text-2xl flex justify-center items-center ml-3"
```
Where is this class defined?